### DISCUSSION TITLE
Considering Shelter Manager for a foster based rescue

### DISCUSSION THREAD

**Discussion by hg303** [sheltermanager/asm3#1375] [Q&A / Help] [answered]
We are a somewhat small shelter based rescue in the US. We currently use spreadsheets but are outgrowing that.  Shelter manager is on our short list software to consider.  I can see many of the functions in the online demo but not the application review process.

I understand that application forms can be uploaded from our website.  But I'm unable to see, in the demo, the review workflow.  How reviewers can enter their review notes (i.e.  the results of their conversations with references, their landlord approval, etc).  Then, how those applications are reviewed for final approval.  Finally, how an adoption coordinator would see the recent approvals in order to take action on them.

Is anyone using Shelter Manager for application review?  If so, can you describe your workflow for this process?  Thanks in advance.

**Comment by Tetzor** [comment, marked as answer]
Hi,

On submission the application forms generate reservations in the reservation book, an email of the application can also be sent automatically to the adoption coordinator. 

Notes can be added to the reservation and the status drop down updated as the application progresses, you can define your own reservation statuses in your lookup data. The reservation book can be filtered on columns such as status and coordinator to allow them to view and manage their work.

https://sheltermanager.com/repo/asm3_help/onlineform.html#application-forms
https://sheltermanager.com/repo/asm3_help/movements.html#reservations
https://sheltermanager.com/repo/asm3_help/movements.html#movement-books
https://sheltermanager.com/repo/asm3_help/configuration.html#lookup-data

Kind regards,

Jon

**Reply by hg303** [reply]
Thank you.  Will look through this.

**Reply by Mdeprima64** [reply]
Yes, we are a foster based rescue and I've introduced ASM with 2 other rescues as well.  I've found it's best to pick which functions add the most value for you first and add on from there.
